### DISCUSSION TITLE
arm64 artifact

### DISCUSSION THREAD

**Discussion by colinw-smartrg** [cilium/pwru#204] [Q&A] [answered]
I would really love to test the new filter-pcap feature. 
My target is arm64, I don't have GO set to build currently.
I see the artifact for x86_64 in the Built and Test. 
Is there a way to get an arm64 artifact like in the Release ?

**Comment by brb** [comment]
You can run `make release` to build arm64 and amd64 in a Docker container. However, from a quick look, it doesn't seem that we cross-compile libpcap.a for arm64. We can fix this next week.

**Comment by brb** [comment, marked as answer]
Once https://github.com/cilium/pwru/pull/219 is merged, you can run `make release` to build for arm64.
